src/Enzo/enzo-core/EnzoConfig.hpp

```diff
@@ -12,6 +12,10 @@
 #define MAX_FIELDS      30
 #define MAX_FILE_GROUPS 10
 
+// we need to include this for the declaration of GrackleChemistryData
+// -> after GitHub PR #342 is merged, we can delete this line
+#include "Enzo/chemistry/GrackleChemistryData.hpp"
```

Comment:
I think this line can now be safely deleted, correct?

Reply:
Yup! You're absolutely right! Thanks for catching this! I removed it. After the tests pass, we can merge